src/Deeson/WardenDrupalBundle/Document/SiteDrupalDocument.php

```diff
@@ -42,11 +42,13 @@ public function setSiteId($siteId) {
    * @return string
    */
   public function getTypeImagePath() {
-    if ($this->getCoreVersion() < 8) {
-      return 'bundles/deesonwardendrupal/images/drupal7-logo.png';
-    }
+    // Grab first digits before first dot
+    preg_match("/^(\d)*?\./", $this->getCoreVersion(), $matches);
```

Comment:
I suspect it fails with Drupal 10, because the regex would capture only `0`, skipping `1`.
I suggest to use `/^(\d+)\./`. It's also easier to read.
Another alternative would be:
```
$version = $explode('.', $this->getCoreVersion())[0];
$version = empty($version) ? 8 : $version;
```